BrainPortal/app/models/cluster_task.rb

```diff
@@ -732,6 +732,8 @@ def setup_and_submit_job
       self.status == 'Setting Up'
     return false if self.workdir_archived?
 
+    self.tool_config.validate_overlays_specs if self.tool_config  # validate and sync files on setup
```

Comment:
What does that even do? It doesn't do anything. And if it crashes, it will crash the entire Bourreau Worker.

Validation methods are part of the database-side life cycle of the ActiveRecord objects, and we're not at all in any sort of code that is part of that life cycle.

Reply:
the requirements included 'sync userfile on setup', is there a more appropriate place to do it?

Reply:
well, removed, maybe redundant indeed